test_files/jsdoc_types.untyped/module2.ts

```diff
@@ -3,8 +3,7 @@ export class ClassTwo {}
 export interface Interface { x: number }
 export class ClassWithParams<T> {}
 
-// TODO(evanm):
-// export type TypeAlias = number;
-// export type TypeAliasWithParam<T> = T[];
```

Comment:
Did you fix these?  How is this related to transformers?

Reply:
I didn't fix these in this change, they've been fixed a long time ago. I just came across the commented out lines here and cleaned it up. I don't think it's worth extracting a separate commit for this.